src/browzine-summon-adapter.js

```diff
@@ -61,11 +66,11 @@ angular.module("summonApp.directives")
   };
 
   function getData(response) {
-    return Array.isArray(response.data.data) ? response.data.data[0] : response.data.data;
+    return Array.isArray(response.data) ? response.data[0] : response.data;
```

Comment:
So does `http.get` do something a lot different than `$.getJSON` that we don't have to access things through `.data.data`?  Just curious.

Reply:
Yeah, I was surprised too. Looks like Angular's `http.get` was wrapping our Public API response in a `data` object. Both awkward and surprising it would modify another service's http response. The base jQuery AJAX calls it extends don't do that.

Reply:
Weird! Nice that we can get rid of that weirdness.